src/Autotype/__init__.py

```diff
@@ -0,0 +1,10 @@
+from importlib import resources
+
+try:
+    import tomllib
```

Comment:
tomllib or tomli not included in pyproject.toml as dependencies. Doing an editable install locally is failing.